### DISCUSSION TITLE
Categories/Tags

### DISCUSSION THREAD

**Discussion by Azzarox** [dailydotdev/daily#738] [Feature Requests]
I think it would be beneficial to the readers if there is better UX/UI to read separated categories instead of going to My Feed icon then Manage Tags and then searching for a specific tag let's say '#crypto' and even then not loading every time (I reported a bug). Personally, would be easier to have another tab in the sidebar with tags. Why would I think this will be beneficial? Well, for example I'm doing mainly web development but in my free time I'm curious about blockchain. However, my main focus is web development, so I don't want in 'my-feed' to have 'blockchain' articles. I use daily.dev mainly for knowledge, inspiration, tips and tricks, etc. which could be of use in my projects.

**Comment by idoshamun** [comment]
Hi @Azzarox,

Thanks for the suggestion. I'll log it down :)

**Comment by msamgan** [comment]
also, this can be updated to squads, categorizing the posts in a squad for better understanding.

**Comment by idoshamun** [comment]
@Azzarox I'm happy to share that we recently launched Custom Feeds which allows you to create different feeds based on your interest
